api_tests/collections/test_views.py

```diff
@@ -1,174 +1,184 @@
 
+@pytest.fixture()
+def user_one():
+    return AuthUserFactory()
 
-class TestCollectionList(ApiTestCase):
-    def setUp(self):
-        super(TestCollectionList, self).setUp()
-        self.user_one = AuthUserFactory()
-        self.user_two = AuthUserFactory()
-        self.deleted_one = CollectionFactory(creator=self.user_one, is_deleted=True)
-        self.collection_one = CollectionFactory(creator=self.user_one)
+@pytest.mark.django_db
+class TestCollectionList:
 
-        self.url = '/{}collections/'.format(API_BASE)
+    @pytest.fixture()
+    def user_two(self):
+        return AuthUserFactory()
 
-    def test_user_one_gets_user_one_collections(self):
-        res = self.app.get(self.url, auth=self.user_one.auth)
-        ids = [each['id'] for each in res.json['data']]
-        assert_not_in(self.deleted_one._id, ids)
-        assert_in(self.collection_one._id, ids)
+    @pytest.fixture()
+    def collection(self, user_one):
+        return CollectionFactory(creator=user_one)
 
-    def test_user_two_gets_nothing(self):
-        res = self.app.get(self.url, auth=self.user_two.auth)
-        ids = [each['id'] for each in res.json['data']]
-        assert_not_in(self.deleted_one._id, ids)
-        assert_not_in(self.collection_one._id, ids)
+    @pytest.fixture()
+    def collection_deleted(self, user_one):
+        return CollectionFactory(creator=user_one, is_deleted=True)
 
-    def test_unauthorized_gets_nothing(self):
-        res = self.app.get(self.url)
+    def test_user_get_own_collections(self, app, url_collection_list, user_one, user_two, collection_deleted, collection):
+
+        #test_user_one_gets_user_one_collections
+        res = app.get(url_collection_list, auth=user_one.auth)
         ids = [each['id'] for each in res.json['data']]
-        assert_not_in(self.deleted_one._id, ids)
-        assert_not_in(self.collection_one._id, ids)
+        assert collection_deleted._id not in ids
+        assert collection._id in ids
 
+        #test_user_two_gets_nothing
+        res = app.get(url_collection_list, auth=user_two.auth)
+        ids = [each['id'] for each in res.json['data']]
+        assert collection_deleted._id not in ids
+        assert collection._id not in ids
 
-class TestCollectionCreate(ApiTestCase):
+        #test_unauthorized_gets_nothing
+        res = app.get(url_collection_list)
+        ids = [each['id'] for each in res.json['data']]
+        assert collection_deleted._id not in ids
+        assert collection._id not in ids
 
-    def setUp(self):
-        super(TestCollectionCreate, self).setUp()
-        self.user_one = AuthUserFactory()
-        self.url = '/{}collections/'.format(API_BASE)
+@pytest.mark.django_db
+class TestCollectionCreate:
 
-        self.title = 'Cool Collection'
+    @pytest.fixture()
+    def user_two(self):
+        return AuthUserFactory()
 
-        self.user_two = AuthUserFactory()
+    @pytest.fixture()
+    def title_collection(self):
```

Comment:
IMO, and from what I have seen in other tests, it's not desirable to have class attributes; we would need use `self.` in test definitions which is more like nose/unittests way. In addition, for maintenance, IMO, it's more desirable to have all/most fixtures listed as params of the test functions. Since pytest fixtures are cached, performance hit of using simple fixtures vs. strings is minimal.